openmdao/utils/om.py

```diff
@@ -442,6 +444,51 @@ def _cite(prob):
     _load_and_exec(options.file[0], user_args)
 
 
+def _show_dependency_versions():
+    import openmdao
+
+    setup_path = os.path.realpath(os.path.dirname(os.path.dirname(openmdao.__file__)))
+    setup_file = os.path.join(setup_path, 'setup.py')
+    if not os.path.exists(setup_file):
+        print('--show_dependencies only works in a development environment.')
+        return
+
+    setup_path = os.getcwd() + "/"
+    if 'openmdao' in setup_path:
+        setup_path = setup_path[:setup_path.index('openmdao')]
```

Comment:
this code can go away now..  replaced by the code above that gets the path from `openmdao.__file__` rather than using `getcwd`